### PR TITLE
Github action to automate additional Community Supporting Processes

### PR DESCRIPTION
**Pull request by RamyaBill**
This is the PR to automate additional community supporting process as detailed in the below Jira issues -

- https://payara.atlassian.net/browse/SERV-1005
- https://payara.atlassian.net/browse/SERV-1006

### CODE REVIEW

**Comment by JamesHillyard** [comment]
Jenkins test please

File: .github/workflows/monitor-inactive-openissues.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+              const updatedAt = new Date(issue.updated_at).getTime();
+              const currentTime = new Date().getTime();
+              const updateMessage = `Please update the issue #${issue.number}.`;
+
+              if (currentTime - updatedAt > timeThreshold) {
+                 await github.rest.issues.createComment({
+                  owner,
+                  repo,
+                  issue_number: issue.number,
+                  body: updateMessage,
```
Comment (fturizo) [inline .github/workflows/monitor-inactive-openissues.yml:45]: As agreed, it is better to replace this action with an MS Teams message instead.

**Review by fturizo — CHANGES_REQUESTED** [review]
Changes needed

**Review by fturizo — APPROVED** [review]
@RamyaBill, I have implemented the changes we discussed.

**Pull request merged**